tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -71,6 +72,7 @@ def test_nodes_restart(
         """
         ocp_nodes = get_node_objs()
         nodes.restart_nodes_by_stop_and_start(nodes=ocp_nodes, force=force)
+        time.sleep(300)
         self.sanity_helpers.health_check()
```

Comment:
Please change `restart_nodes_by_stop_and_start` to pass the timeout value into the function itself.